app/controllers/api/v1/organization_controller.rb

```diff
@@ -36,8 +36,8 @@ def create
 
     name = params[:name]
 
-    if name.nil?
-      render stauts: 200,
+    if name.nil? || name == ""
```

Comment:
Just for the future, ruby has a nice method called .blank? that checks the following: blank? objects are false, empty, or a whitespace string. For example, "", " ", nil, [], and {} are blank.

So this can be simplified to name.blank?